languages/translations/de.json

```diff
@@ -84,23 +84,34 @@
         },
         "continue": {
             "default": {
-                "default": "{modifier} weiterfahren",
-                "name": "{modifier} weiterfahren auf {way_name}",
-                "destination": "{modifier} weiterfahren Richtung {destination}"
+                "default": "{modifier} abbiegen",
```

Comment:
Does "abbiegen" make sense here? It basically means "to turn", but you can get a `continue straight` which we would then translate into "Geradeaus abbiegen" -> "turn straight" (?)

Reply:
Sounds like we need a `continue straight` override like we've done here: https://github.com/Project-OSRM/osrm-text-instructions/blob/master/languages/translations/en.json#L93